app/views/phrasing_phrases/index.html.haml

```diff
@@ -1,9 +1,12 @@
-#search
+#search{ style: "display:inline-block;" }
   = form_tag phrasing_phrases_path, method: :get do
     - locale_options = options_for_select([nil] + @locale_names.map{|l| [l,l]}, params[:locale])
     = select_tag 'locale', locale_options
     = text_field_tag :search, params[:search], placeholder: 'Search for keys or values', size: 40
     = submit_tag 'Search', class: "btn btn-success"
+
+= link_to 'Download', url_for(params.merge(format: 'yaml')), { class: 'btn btn-success', style: "display:inline-block;" }
```

Comment:
same thing for this inline css. You should add a css class to it and reference it from the css file.